src/forms/subscriptions.ts

```diff
@@ -154,6 +154,10 @@ class FormFields extends BaseFormFields {
     //                array of triggers for the current team
     addChannelTrigger(triggers: ZDTrigger[]): void {
```

Comment:
The comment above this says `addChannelTrigger adds the team triggers`. Is it adding channel triggers or team triggers? 🙂